packages/account/src/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -135,6 +137,8 @@ export const PersonalDetailsForm = ({
 
     const isMounted = useIsMounted();
 
+    const should_show_email = is_social_signup || (is_virtual && has_active_real_account);
```

Comment:
show email box in personal details to social accounts(real and virtual).
show email box in personal details to non-social virtual accounts just in case it has at least one real_account